src/nvim/map.c

```diff
@@ -123,29 +123,8 @@
     kh_clear(T##_##U##_map, map->table); \
   }
 
-static inline khint_t String_hash(String s)
```

Comment:
Is there really a point to delete these? It seems not very unlikely that we might want a mutable string hash in the future.